openapi.yml

```diff
@@ -7,7 +7,7 @@ servers:
   - url: /api/kytos/topology
     description: Local server (uses test data)
 paths:
-  /api/kytos/topology/v3/:
+  /v3/:
```

Comment:
Appreciated you proactively taking the opportunity to also enhance this part of the OpenAPI docs.